static/js/tools/map/ready_hooks.ts

```diff
@@ -282,15 +282,18 @@ export function useRenderReady(chartStore: ChartStore) {
   const { display, statVar } = useContext(Context);
   const breadcrumbValueReady = useBreadcrumbValuesReady(chartStore);
   const mapValuesDatesReady = useMapValuesDatesReady(chartStore);
+  const geoJsonReady = useGeoJsonReady(chartStore);
   return useCallback(() => {
     return (
       statVar.value.info &&
+      geoJsonReady() &&
```

Comment:
I think we only need to wait for geoJson before rendering if the map type is D3

Reply:
I added a hook for deciding the map type, which implicitly checks data availability too.

Reply:
I'm not sure about doing the check with this hook because we want to default to d3 map & update to leaflet map if we get raster data back before all the d3map information. With the hook you added, it will update the mapType back to d3 once the d3map information is completed. 

Can we go back to the original way of updating mapType & just pass mapType as a parameter to this ready function and then check that `(geoJsonReady() || mapType === MAP_TYPE.LEAFLET)`?